code/modules/stock_market/articles.dm

```diff
@@ -0,0 +1,129 @@
+		"neutral_outcome" = list("it's not lifting off as expected", "it's not selling according to the expectations", "it's only generating enough profit to cover the marketing and manufacturing costs", "it does not look like it will become a massive success"), \
+		"positive_outcome" = list("it's already sold out", "it has already sold over one billion units", "suppliers cannot keep up with the wild demand", "several companies using this new technology are already reporting a projected increase in profits"), \
+		"resounding" = list("resounding", "tremendous", "total", "massive", "terrific", "colossal"), \
+		"rise" = list("rise", "increase", "fly off the positive side of the charts", "skyrocket", "lift off"), \
+		"sell" = list("sell!", "sell, sell, sell!", "bail!", "abandon ship!", "get out before it's too late!", "evacuate!", "withdraw!"), \
+		"signifying" = list("signifying", "indicating", "displaying the sign of", "displaying"), \
+		"sneak_peek" = list("review", "sneak peek", "preview"), \
+		"stock_market" = list("stock market", "stock exchange"), \
+		"stockholder" = list("stockholder", "shareholder"), \
+		"success" = list("success", "triumph", "victory"), \
+		"this_time" = list("this week", "last week", "this month", "yesterday", "today", "a few days ago") \
+	)
+
+/datum/article/New()
+	..()
+	if ((outlets.len && !prob(100 / (outlets.len + 1))) || !outlets.len)
+		var/ON = generateOutletName()
+		if (!(ON in outlets))
+			outlets[ON] = list()
+		outlet = ON
+	else
+		outlet = pick(outlets)
+
+	var/list/authors = outlets[outlet]
+	if ((authors.len && !prob(100 / (authors.len + 1))) || !authors.len)
+		var/AN = generateAuthorName()
+		outlets[outlet] += AN
+		author = AN
+	else
+		author = pick(authors)
+
+	ticks = ticker.round_elapsed_ticks
+
+/datum/article/proc/generateOutletName()
+	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia")
+	var/list/nouns = list("Post", "Herald", "Sun", "Tribune", "Mail", "Times", "Journal", "Report")
+	var/list/timely = list("Daily", "Hourly", "Weekly", "Biweekly", "Monthly", "Yearly")
+
+	switch(rand(1,2))
+		if (1)
+			return "The [pick(locations)] [pick(nouns)]"
+		if (2)
+			return "The [pick(timely)] [pick(nouns)]"
+
+/datum/article/proc/generateAuthorName()
+	switch(rand(1,3))
+		if (1)
+			return "[consonant()]. [pick(last_names)]"
+		if (2)
+			return "[prob(50) ? pick(first_names_male) : pick(first_names_female)] [consonant()].[prob(50) ? "[consonant()]. " : null] [pick(last_names)]"
+		if (3)
+			return "[prob(50) ? pick(first_names_male) : pick(first_names_female)] \"[prob(50) ? pick(first_names_male) : pick(first_names_female)]\" [pick(last_names)]"
+
+/datum/article/proc/formatSpacetime()
+	var/ticksc = round(ticks/100)
```

Comment:
I don't like that you added an entire tick recording system just for /this/
given that station time is already a thing, you can just fluff around with the value for station time or keep it the same.